tests/unit/test_vid_club.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+from datetime import datetime
+from your_module import VidClub, ValidationError
```

Comment:
Oops 😄

Reply:
hm, it's still wrong. Should be `from viadot.sources import VidClub, ValidationError`

Reply:
Done